include/cuco/static_map_ref.cuh

```diff
@@ -227,11 +227,7 @@ class static_map_ref
   [[nodiscard]] __host__ __device__ constexpr auto probing_scheme() const noexcept;
 
   /**
-   * @brief Creates a reference with new operators from the current object.
-   *
-   * @deprecated This function is deprecated. Use the new `with_operators` instead.
```

Comment:
Are we dropping the deprecation warning?

Reply:
I removed this API directly since it's marked as deprecated for a while and no exiting RAPIDS code uses it.